pkg/integrations/v2/app_o11y_receiver/app_o11y_receiver.go

```diff
@@ -86,38 +91,56 @@ func (c *Config) NewIntegration(l log.Logger, globals integrations.Globals) (int
 		return nil, err
 	}
 	sourcemapLogger := log.With(l, "subcomponent", "sourcemaps")
-	sourcemapStore := sourcemaps.NewSourceMapStore(sourcemapLogger, c.ExporterConfig.SourceMaps, reg, nil, nil)
-
-	logsInstance := globals.Logs.Instance(c.ExporterConfig.LogsInstance)
-	lokiExporter := exporters.NewLokiExporter(
-		l,
-		exporters.LokiExporterConfig{
-			LokiInstance:     logsInstance,
-			Labels:           c.ExporterConfig.LogsLabels,
-			SendEntryTimeout: c.ExporterConfig.LogsSendTimeout,
-		},
-		sourcemapStore,
-	)
+	sourcemapStore := sourcemaps.NewSourceMapStore(sourcemapLogger, c.ReceiverConfig.SourceMaps, reg, nil, nil)
 
 	receiverMetricsExporter := exporters.NewReceiverMetricsExporter(exporters.ReceiverMetricsExporterConfig{
 		Reg: reg,
 	})
 
 	var exp = []exporters.AppO11yReceiverExporter{
-		// Loki
-		lokiExporter,
 		receiverMetricsExporter,
 	}
 
-	handler := handler.NewAppO11yHandler(c.ExporterConfig, exp, reg)
+	if len(c.ReceiverConfig.LogsInstance) > 0 {
```

Comment:
Ι think we should make both the logs and the tracing instances hard dependencies (i.e. if the integration is enabled and the instances are missing for some reason, exit), or have default fallbacks

Reply:
Hm, I'm not certain that every user will have both logging and tracing backends and want to use them with frontend, would rather allow flexibility. It doesn't cost us anything